index.html

```diff
@@ -2316,11 +2333,10 @@ <h3>Global State</h3>
                   <li><p>Make the <a>session timeouts</a> the new <var>timeouts</var>.
             </ol>
 
-            <li><p>Set a property on <var>capabilities</var> with name
-                "<code>timeouts</code>" and value that of the
-                <a>JSON deserialization</a> of the <a>session timeouts</a>.
+        <li><p>Set a property on <var>capabilities</var> with name
+          "<code>timeouts</code>" and value that of the
+          <a>JSON deserialization</a> of the <a>session timeouts</a>.
```

Comment:
I assume that this should probably be part of the other PR about timeouts, right?